website/static/js/profile.js

```diff
@@ -647,12 +664,33 @@ var ListViewModel = function(ContentModel, urls, modes) {
 ListViewModel.prototype = Object.create(BaseViewModel.prototype);
 
 ListViewModel.prototype.addContent = function() {
-    this.contents.push(new this.ContentModel(this));
+    if (!this.institutionObjectsEmpty() && this.isValid()) {
+        this.contents.push(new this.ContentModel(this));
+    }
+    else {
+        this.changeMessage(
+            'Institution Field is Required',
```

Comment:
Sentence case is preferred: https://centerforopenscience.github.io/osf-style/language.html#capitalization